packages/types/src/errors.ts

```diff
@@ -2,6 +2,7 @@
 // See LICENSE.txt for license information.
 
 export type ServerError = {
+    type?: string;
```

Comment:
This is actually used in a few different places (eg. the AnnouncementBanner component), but those either haven't been migrated to TS or they use `any` for their type